bundles/net.tourbook/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -1,3 +1,4 @@
+cleanup.add_all=false
```

Comment:
@wolfgang-ch 
I have updated this file and it resolved all the non desired code optimizations that were added [in 2021.06 (4.20)](https://www.eclipse.org/eclipse/news/4.20/jdt.php#use-instanceof)

Reply:
You have to update this file in all MT plugins, see commit comment in https://github.com/wolfgang-ch/mytourbook/commit/72b6585e541c078ec4bad6a69f7ebedf9da1e1ba how to do it easily

Reply:
Ok, i thought this file was used by all the other plugins. I now know it for the next time Eclipse adds cleanups.